src/styles/components/_product-card.scss

```diff
@@ -0,0 +1,105 @@
+.bsds-product-card
+{
+
+  a[href]::after,
+  a[href]::before
+  {
+    content: "";
+    display: block;
+    position: absolute;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+  }
+
+  .bsds-product-card-ns-title
+  {
+    @extend .bsds-link;
+    margin: 0;
+    width: fit-content;
+
+    &[href]
+    {
+      text-decoration: none;
+
+      &:hover
+      {
+        @extend .bsds-link;
+      }
+    }
+  }
+
+  .bsds-card-title
+  {
+    [class*="bsds-link"]
+    {
+      text-decoration: none;
+
+      &:hover
+      {
+        @extend .bsds-link;
+      }
+    }
+  }
+
+  img
+  {
+    ~.bsds-card-title,
+    ~.bsds-product-card-ns-title
+    {
+      margin-top: space(md);
+    }
+  }
+}
+
+.bsds-card-border-light
+{
+  &.bsds-product-card
+  {
+    padding: 0;
+
+    [class*="bsds-tag"]
+    {
+      padding: space(3x) space(3x) 0 space(3x);
+
+      ~[class*="bsds-card-title"],
+      ~[class*="bsds-product-card-ns-title"]
+      {
+          margin-top: space(lg);
+      }
+    }
+
+    [class*="bsds-card-title"],
+    [class*="bsds-product-card-ns-title"]
+    {
+      margin: space(3x) space(3x) 0;
+    }
+
+    .bsds-card-description
+    {
+      padding: 0 space(3x) space(3x) space(3x);
+    }
+  }
+}
+
+[class*="bsds-tag"]
+{
+  ~ .bsds-product-card-ns-title
+  {
+    margin-top: space(lg);
+  }
+}
+
+.bsds-product-card-with-image
```

Comment:
Can we name this to something more specific to its use case since it isn't used on all cards with images?